src/main/resources/wa-task-initiation-employment-et_scotland.dmn

```diff
@@ -2179,6 +2179,329 @@ else null</text>
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0t9utpy">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0lpd561">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_050vfln">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1hbffqf">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0rujxtm">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_09002lk">
+          <text>"Amend claim",
+"Consider decision afresh",
+"Contact the tribunal",
+"Order a witness to attend to give evidence",
+"Order other party",
+"Postpone a hearing",
+"Reconsider judgment",
+"Respondent not complied",
+"Restrict publicity",
+"Strike out all or part of the response",
+"Vary or revoke an order"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_07pv6ht">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0jdrode">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_03o4js8">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0ru3kip">
+          <text>"ContactTribunalWithAnApplication"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0ksvdqw">
+          <text>LATEST_APPLICATION_TASK_NAME</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_082s2a6">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1ldurlx">
+          <text>"Application"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1hsijlk">
+        <inputEntry id="UnaryTests_0g31tjl">
+          <text>"tseClaimantRepResponse"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1qmt6av">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1lsu9qk">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1iaabbh">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1uycd46">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0iyiowu">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0gjulca">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1eg88pd">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0n73org">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_087xdl8">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1ywj5qo">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0q8352v">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1qm6fe3">
+          <text>"Claimant"</text>
```

Comment:
Noticed this when reviewing callbacks - the response from is set to Claimant Representative and not Claimant so this task might not trigger - https://github.com/hmcts/et-ccd-callbacks/pull/2259/files#diff-f007255afc0d633a2c2099ce178ce90cd4e16e7f2684a76a8d24177611e13700R211

Reply:
addressed comments [e95be0d](https://github.com/hmcts/et-wa-task-configuration/pull/263/commits/e95be0d9a145b0fd0fe51e0b1a9ff52b125285c6)